regression/benchmark/result_template_sysbench.json

```diff
@@ -0,0 +1,26 @@
+[
+    {
+        "name": "Threads Minimum latency",
+        "unit": "ms",
+        "value": 0,
+        "extra": "min"
+    },
+    {
+        "name": "Threads Average Latency",
+        "unit": "ms",
+        "value": 0,
+        "extra": "avg"
+    },
+    {
+        "name": "Threads Maximum Latency",
+        "unit": "ms",
+        "value": 0,
+        "extra": "max"
+    },
```

Comment:
So, are we going to compare all the recorded results here? I think outcomes like `max`, `min` might be somewhat random, even if you've set a relatively large threshold. I suggest that comparing the `avg` results would be sufficient for now.

Reply:
For current version, the answer is yes. `sysbench` gives `max`, `min`, `avg` and `per95`. Maybe the latter two are more suitable to be set up for comparison.